ibm_catalog.json

```diff
@@ -96,6 +109,66 @@
               },
               "key": "region",
               "required": true
+            },
+            {
+              "key": "existing_secrets_manager_instance_crn"
+            },
+            {
+              "key": "existing_secrets_manager_cert_crn"
+            },
+            {
+              "key": "cert_common_name"
+            },
+            {
+              "key": "root_ca_name"
+            },
+            {
+              "key": "root_ca_common_name"
+            },
+            {
+              "key": "intermediate_ca_name"
+            },
+            {
+              "key": "certificate_template_name"
+            },
+            {
+              "key": "existing_subnet_names"
+            },
+            {
+              "key": "vpn_subnet_cidr_zone_1"
+            },
+            {
+              "key": "vpn_subnet_cidr_zone_2"
+            },
+            {
+              "key": "vpn_client_access_group_users"
+            },
+            {
+              "key": "access_group_name"
+            },
+            {
+              "key": "create_policy"
+            },
+            {
+              "key": "vpn_server_routes"
+            },
+            {
+              "key": "existing_vpc_crn"
```

Comment:
There might be a custom catalog type for VPC. Can you check? (as part of https://github.ibm.com/GoldenEye/issues/issues/11555)

Reply:
as discussed with catalog team, crn is not supported by the picker